### DISCUSSION TITLE
Support for MySQL

### DISCUSSION THREAD

**Discussion by darrellme** [Apicurio/apicurio-registry#2505] [Q&A] [answered]
Hi

Are there any plans to support MySQL for the API registry?

Thanks

Darrell Merryweather

**Comment by EricWittmann** [comment, marked as answer]
No plans currently.  However, if someone were interested in contributing mysql support it would probably not be very hard.  The DB schema we use is pretty simple - most of the DDL and SQL queries will probably work as-is.  I'd be happy to help with guidance and answering questions if anyone wanted to try.

Probably the most annoying thing is that, due to how Quarkus works, support for mysql likely requires a separate maven module.  Because DB support is configured at **build** time for Quarkus.
